lib/db.js

```diff
@@ -973,12 +974,4 @@ Db.prototype.getLogger = function() {
  * @type {Db}
  */
 
-// Constants
```

Comment:
By removing this, it would technically break our public API, since these values were previously exposed publicly. Can we add the following code to ensure they are still exposed?

```js
Object.assign(Db, dbConstants);
```

Reply:
Yes, interesting approach.